### PR TITLE
Revert "Create white glove AB test" - Resolving the revert conflicts

### PR DESCRIPTION
**Pull request by southp**
#### Changes proposed in this Pull Request

This PR is part of the effort of reverting all the changes introduced by #42783. 
For the reverting strategy applied here, please refer to the base branch: https://github.com/Automattic/wp-calypso/pull/44022

#### Testing instructions

* Code review. Make sure the conflict resolution makes sense.
* Make sure you won't see the Expert Guidance offer even if you explicitly assign yourself to the variation group. There is an accompanied Playwright script below that can automate this.
* Run through a few purchasing flow and make sure they work.

#### Run the test script

1. Copy the script below into a file and put it in a folder. e.g. `sandbox/expert-guidance.js`
1. In `sandbox`, run `npm i -D playwright`
1. Run the script by `node expert-guidance.js`

On the master branch, running this script should lead you to the checkout page with the Expert Guidance product in the cart; on this branch, it shouldn't.

```
const { chromium } = require( 'playwright' );
const assert = require( 'assert' );

( async () => {
	const browser = await chromium.launch( {
		headless: false,
	} );
	const context = await browser.newContext( {
		locale: 'en',
	} );
	const page = await context.newPage();

	await page.goto( 'http://calypso.localhost:3000/start' );
	await page.evaluate( () => {
		window.localStorage.setItem( 'ABTests', '{"whiteGloveUpsell_20200623":"variantShowOffer"}' );
	} );

	const randSerial = Math.floor( 100000 * Math.random() ) + 65535;
	const email = 'southptest2+' + randSerial + '@gmail.com';
	const userName = 'southptest' + randSerial;
	const password = <fill in whatever you want>

	// user step
	await page.fill( 'css=input#email', email );
	await page.fill( 'css=input#username', userName );
	await page.fill( 'css=input#password', password );
	await page.click( 'css=button[type="submit"]' );

	await page.waitForNavigation();

	// domain step
	const freeDomain = userName + '.wordpress.com';
	await page.fill( 'css=input[type="search"]', freeDomain );
	await page.click( 'css=div[data-e2e-domain="' + freeDomain + '" >> css=button[type="button"]' );

	await page.waitForNavigation();

	// plan step
	await page.click( 'css=.plans-features-main__banner-content >> css=button[type="button"]' );
	await page.waitForNavigation( { url: /offer-white-glove/ } );

	// expert guidance page
	const buyButtonSelector = 'css=button.white-glove__accept-offer-button';
	await page.waitForSelector( buyButtonSelector );
 	await page.click( buyButtonSelector );
} )();

```

### CODE REVIEW

File: client/my-sites/checkout/upsell-nudge/index.jsx [hunk 1]
```diff
@@ -166,20 +166,7 @@ export class UpsellNudge extends React.Component {
 		}
 	}
 
-<<<<<<< HEAD
-	getUrl( url, args ) {
```
Comment (southp) [inline client/my-sites/checkout/upsell-nudge/index.jsx:170]: This function isn't used anywhere, hence removing it.

**Comment by sirbrillig** [comment]
For the composite checkout files, you're likely not going to be able to easily apply a revert since so much has changed since the original code was added. It might be better to search for `WhiteGlove` in the code instead.

**Comment by southp** [comment]
@sirbrillig Thanks for taking a look. You are right that there has been so many changes that make a direct revert impossible. That is why I'd like to make the best use of the mechanical reverting commit while keeping the best visibility of the conflict resolution. Please refer to the base branch for the reverting strategy here: https://github.com/Automattic/wp-calypso/pull/44022. And my sincere apology that I forgot to include it in the description in the first place and all the confusion it has caused :( 

Does this make sense to you?

**Comment by sirbrillig** [comment]
Ah, yes, sorry I didn't think to look at the base branch! Please let me know if I can help (although I will be afk for the next five days). If it's useful, I went through and just removed the white glove code from composite checkout in this branch: https://github.com/Automattic/wp-calypso/pull/new/remove/white-glove-composite-checkout  Feel free to use it if you want!

File: client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js [hunk 2]
```diff
@@ -58,17 +55,11 @@ function WPLineItem( {
 	const shouldShowVariantSelector = getItemVariants && item.wpcom_meta && ! isRenewal;
 	const isGSuite = isGSuiteProductSlug( item.wpcom_meta?.product_slug );
 
-<<<<<<< HEAD
 	const productSlug = item.wpcom_meta?.product_slug;
 	const isBusinessPlanProduct = productSlug && isBusinessPlan( productSlug );
```
Comment (aneeshd16) [inline client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js:59]: i think it should be safe to remove `productSlug` and `isBusinessPlanProduct` as well.

Comment (aneeshd16) [reply client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js:59]: I've had a look at the conflict resolution, looks good to me 👍

Comment (southp) [reply client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js]: Nice catch. I have removed them along with a few more unused symbols in f672950.

**Pull request merged**